libs/books/data-access/src/lib/+state/reading-list.actions.ts

```diff
@@ -39,3 +39,17 @@ export const removeFromReadingListFailed = createAction(
 export const removeFromReadingListSucceeded = createAction(
   '[Reading List API] Confirmed remove from list'
 );
+
+export const markBookAsFinished = createAction(
+  '[Reading List API] Mark book as finished',
+  props<{ bookId: string, finishedDate: string }>()
+);
+
+export const markBookAsFinishSucceeded = createAction(
+  '[Reading List API] Confirmed mark book as finished'
+);
+
+export const markBookAsFinishFailed = createAction(
```

Comment:
`markBookAsFinishedFailed` for consistency

Reply:
I have updated the action name.